cloud/ec2.go

```diff
@@ -1489,10 +1489,17 @@ func (m *ec2Manager) CreateVolume(ctx context.Context, volume *host.Volume) (*ho
 			{ResourceType: aws.String(ec2.ResourceTypeVolume), Tags: volumeTags},
 		},
 	}
-	// Iops is required for io1.
-	if volume.Type == VolumeTypeIo1 {
```

Comment:
so short lived 😂  can we still keep the comment though

Reply:
ah I meant to keep it and then forgot about it when I couldn't figure out where to put it.
An inline comment? Or inside the else if?

Reply:
both work, this looks good to me!